lib/conversation.py

```diff
@@ -67,6 +67,7 @@ def command(self, line: ChatLine, cmd: str) -> None:
         :param cmd: The command to react to.
         """
         from_self = line.username == self.game.username
+        is_eval = cmd.startswith("eval")
         if cmd == "commands" or cmd == "help":
             self.send_reply(line, "Supported commands: !wait (wait a minute for my first move), !name, !howto, !eval, !queue")
```

Comment:
I think it would be good to replace the string here with something like: "Supported commands: !wait (wait a minute for my first move), !name, !howto, !eval (or any text starting with !eval), !queue"

Make sure the resulting string is less than 140 characters.

Reply:
Done!